src/rust/engine/workunit_store/src/lib.rs

```diff
@@ -68,6 +70,7 @@ impl WorkunitMetadata {
   pub fn new() -> WorkunitMetadata {
     WorkunitMetadata {
       display: true,
```

Comment:
So... should we get rid of `display: bool` at the same time? Feels like it might be straightforward to do, and this PR is pretty small.